src/payment/strategies/adyenv2/adyenv2-initialize-options.ts

```diff
@@ -0,0 +1,25 @@
+import { CardComponentOptions } from './adyenv2';
+
+/**
+ * A set of options that are required to initialize the AdyenV2 payment method.
+ *
+ * Once AdyenV2 payment is initialized, credit card form fields, provided by the
+ * payment provider as iframes, will be inserted into the current page. These
+ * options provide a location and styling for each of the form fields.
+ */
+export default interface AdyenV2PaymentInitializeOptions {
+    /**
+     * The location to insert the Adyen component.
+     */
+    containerId: string;
+
+    /**
+     * Optional. Overwriting the default options
+     */
+    options?: CardComponentOptions;
```

Comment:
I think you'll need to be a little bit more explicit here to prevent problems in the future. As far as I know, there are two types of components you could potentially create via this strategy - the credit card component and the 3DS widget. This field is for the card component. And the field below (`threeDS2ChallengeWidgetSize`) is for the 3DS widget. For future-proofing, I suggest you to be explicit about the two options - have a `cardOptions` and `threeDS2Options`.